adm/src/main.rs

```diff
@@ -107,9 +107,11 @@ fn parse_args<'a>() -> ArgMatches<'a> {
             (@arg quiet: -q --quiet "do not display output"))
         (@subcommand genesis =>
             (about: "creates the genesis.batch file for initializing the validator")
-         (@arg input_file:
-          +takes_value ... "file or files containing batches to add to the resulting")
-            (@arg output: -o --output "choose the output file for GenesisData"))
+            (@arg input_file:
+             +takes_value ... "file or files containing batches to add to the resulting")
+            (@arg output: -o --output "choose the output file for GenesisData")
+            (@arg ignore_required_settings: --ignore_required_settings
```

Comment:
This should be `--"ignore-required-settings"` (no underscores in CLI flags)

Reply:
Done